bundle/config/mutator/translate_paths_jobs.go

```diff
@@ -50,6 +50,11 @@ func rewritePatterns(t *translateContext, base dyn.Pattern) []jobRewritePattern
 			t.translateNoOp,
 			noSkipRewrite,
 		},
+		{
+			base.Append(dyn.Key("libraries"), dyn.AnyIndex(), dyn.Key("requirements")),
+			t.translateNoOp,
```

Comment:
You have to do the translation here and not skip it

Reply:
Done, please review